mindjunkies/dashboard/views.py

```
@@ -48,3 +62,49 @@
+
+        # Handle certificate files and save them
+        certificates = self.request.FILES.getlist('certificates')  # Get multiple files
+
+        teacher_verification.save()
+    
+        for certificate in certificates:
+            cert_instance = Certificate.objects.create(image=certificate)
+            # Add the saved certificate instance to the ManyToManyField
+            teacher_verification.certificates.add(cert_instance)
+
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)